joomla/joomla-cms
# [4.0] correcting com_content choose layout field class

### Summary of Changes
Adding class `custom-select` to all com_content fileds of type `componentlayout`


### Testing Instructions
Check all places where Change Layout or Layout is proposed in com_content stuff
Global com_content Options
Article edit options

Menu items:
List all categories
Category blog
category list
### Before patch
<img width="474" alt="screen shot 2018-08-26 at 08 21 25" src="https://user-images.githubusercontent.com/869724/44625605-9b4d3f00-a90d-11e8-9a8f-c3abb47d921f.png">


### After patch
<img width="511" alt="screen shot 2018-08-26 at 08 24 04" src="https://user-images.githubusercontent.com/869724/44625608-af913c00-a90d-11e8-9398-6d1fc4a091f9.png">


NOTE: Looks like there are some issues here with `uitab` when editing a menu item (firefox but also Safari f). Therefore it may be difficult to get to the tab where the field concerned is displayed.
